CHANGELOG.md

```diff
@@ -68,12 +68,13 @@ Ref: https://keepachangelog.com/en/1.0.0/
 ### API Breaking
 
 * Fix for missing validation for marker permissions according to marker type.  Markers of type COIN can no longer have
-  the Transfer permission assigned.  Existing permission entries on Coin type markers of type Transfer are removed 
+  the Transfer permission assigned.  Existing permission entries on Coin type markers of type Transfer are removed
   during migration [#428](https://github.com/provenance-io/provenance/issues/428)
 
 ### Improvements
 
 * Updated to Cosmos SDK Release v0.43  [#154](https://github.com/provenance-io/provenance/issues/154)
+* Updated wasmd for Cosmos SDK Release v0.43 [#409](https://github.com/provenance-io/provenance/issues/409)
```

Comment:
Should we add a sub bullet here that indicates which version of provwasm, cosmwasm, and the wasmer vm are associated with this version?

Reply:
maybe link to release notes of prov-wasmd

Reply:
I think the sub bullets will be a good addition instead of a link.  That way it will keep the user on our release notes instead of directing them away to another site.